Module-3/Homework.playground/Contents.swift

```diff
@@ -0,0 +1,211 @@
+import Foundation
+/*: ## Part 2 - Programming assignment.
+ */
+/*:
+ a) Create an array of `Int` called **nums** with values 0 through 20. Iterate over the `Array` and print the even numbers.
+ */
+let nums = Array(0...20)
+for num in nums where num % 2 == 0 {
+    print(num)
+}
+/*:
+ b) In your Playground, define the following String:
+ 
+ `let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"`
+ 
+ Iterate over sentence counting the vowels (a, e, i, o, u), ignoring the case.
+ */
+let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"
+let vowels: [Character] = ["a", "e", "i", "o", "u"]
+var count = 0
+for character in sentence.lowercased() {
+    if vowels.contains(character) { count += 1 }
+}
```

Comment:
Perfectly fine and readable, but for the reference, we can make this even shorter. For example, by using `filter` method on an array and then counting the elements.

Reply:
Yes, filter would make this much cleaner.